shell/platform/windows/client_wrapper/BUILD.gn

```diff
@@ -79,6 +79,8 @@ executable("client_wrapper_windows_unittests") {
     "plugin_registrar_windows_unittests.cc",
   ]
 
+  defines = [ "FLUTTER_DESKTOP_LIBRARY" ]
```

Comment:
This flips the export macros in flutter_windows.h from import to export, which fixes the warnings about importing a locally-defined symbol when compiling these unit tests. This was an opportunistic fix for a pre-existing issue, since having compilation runs that changed just the unit tests while working on this reminded me I needed to fix it.

Reply:
Gotcha.  Worth a comment to that effect?

Reply:
Good idea; done.